tic1.js

```diff
@@ -0,0 +1,127 @@
+// this is a test
+const readline = require('readline');
+
+const readlineInterface = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout
+});
+
+function ask(questionText) {
+    return new Promise((resolve, reject) => {
+      readlineInterface.question(questionText, resolve);
+    });
+  }
+
+let cell1 = 1
+let cell2 = 2
+let cell3 = 3
+let cell4 = 4
+let cell5 = 5
+let cell6 = 6
+let cell7 = 7
+let cell8 = 8
+let cell9 = 9
+let cells = [cell1, cell2, cell3, cell4, cell5, cell6, cell7, cell8, cell9]
+let divider = '--- --- ---'
+
+let printGrid = function() {
+    console.log(('\n '+cells[0]+' | '+cells[1]+' | '+cells[2]+'\n'+divider+
+    '\n '+cells[3]+' | '+cells[4]+' | '+cells[5]+'\n'+divider+
+    '\n '+cells[6]+' | '+cells[7]+' | '+cells[8]))
+    }
+
+let askForInputX = async function(){
+    let input = await ask('>');
+    if (cells[parseInt(input)-1] == 'X' || cells[parseInt(input)-1] == 'O'){
+        console.log("\nSorry! That space has been claimed! Try again.\n");
+        askForInputX();
+    }else{
+        for (let item of cells){
+            if (item === parseInt(input)){
+                cells[parseInt(input)-1] = 'X'
+            }
+        }
+        winCheck(cells);
+        playerO();

```

Comment:
This double-recursive call is clever but makes your program logic a bit tangled. It'd probably be better to handle "whose turn is next" in a separate function, that calls `askForInputO` or `askForInputX` as appropriate, then calls `winCheck`, so the `askForInput` functions are responsible for fewer tasks.